src/Common/ErrorResponseModel.cs

```diff
@@ -12,19 +12,10 @@ namespace Vasont.Inspire.Models.Common
     /// </summary>
     public class ErrorResponseModel
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="ErrorResponseModel"/> class.
-        /// </summary>
-        public ErrorResponseModel()
-        {
-            this.Messages = new List<ErrorModel>();
-            this.HasUnhandledException = false;
```

Comment:
False is default below then?

Reply:
False is always default for a bool.

Reply:
Just like 0 is always default for an int. I took that out in another place as well.